lib/FundDistribution/FundDistributionFields/FundDistributionFieldsContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import { getFundsForSelect } from '../../utils';
+import { baseManifest } from '../../manifests';
+import FundDistributionFields from './FundDistributionFields';
+
+const FundDistributionFieldsContainer = ({
+  resources, formValues, disabled, required, totalAmount, currency,
+}) => {
+  const funds = getFundsForSelect(resources);
+
+  return (
+    <FundDistributionFields
+      formValues={formValues}
+      funds={funds}
+      totalAmount={totalAmount}
+      required={required}
+      disabled={disabled}
+      currency={currency}
+    />
+  );
+};
+
+FundDistributionFieldsContainer.manifest = Object.freeze({
+  funds: {
```

Comment:
I'm not sure about the name. We might end up in situation like with acqUnits. When name of resource is the same, but queries are different and they rewrite records of each other. Since it's a common component it might make sense to use some specific value, or at least more obvious, like `dictFunds` from Dictionary Funds (not filtered). And it would be great to have its name as a constant since you use the same "magiv" string in getFundsForSelect